langchain4j-azure-open-ai/src/main/java/dev/langchain4j/model/azure/InternalAzureOpenAiHelper.java

```diff
@@ -45,7 +45,7 @@ class InternalAzureOpenAiHelper {
 
     private static final Logger logger = LoggerFactory.getLogger(InternalAzureOpenAiHelper.class);
 
-    public static final String DEFAULT_USER_AGENT = "langchain4j-azure-openai";
+    public static final String DEFAULT_USER_AGENT = "LangChain4j";
```

Comment:
@jdubois would it be ok if we change this or better not?

Reply:
@langchain4j I'm using it for reporting purposes on the Azure OpenAI side: if you do this, we'll lose the history and we'll need to update to use the new key. Also, the idea was to differentiate "LangChain4J Azure OpenAI" and "LangChain4J OpenAI", as both are able to use Azure OpenAI and I would like to know which LangChain4J module people are using (so I can prioritise work).

Reply:
ok, no worries

Reply:
@roytruelove could you please rollback changes to Azure OpenAI and OpenAI?